edb/server/protocol/ai_ext.py

```diff
@@ -1181,18 +1194,51 @@ async def _start_openai_like_chat(
     model_name: str,
     messages: list[dict[str, Any]],
     stream: bool,
+    temperature: Optional[float],
+    top_p: Optional[float],
+    max_tokens: Optional[int],
+    seed: Optional[int],
+    safe_prompt: Optional[bool],
+    logit_bias: Optional[dict[int, int]],
+    logprobs: Optional[bool],
+    user: Optional[str],
+    tools: Optional[list[dict[str, Any]]]
 ) -> None:
+    isOpenAI = "openai" in str(getattr(client, "base_url", ""))
+    params = {
+        "model": model_name,
+        "messages": messages,
+        **({"temperature": temperature} if temperature is not None else {}),
```

Comment:
This would probably be easier to read if you just used a plain `if`:

```python
params = {
    "model": model_name,
    "messages": messages,
}
if temperature is not None:
    params["temperature"] = temperature
...
```